libs/pando/ui-sheet/src/sheet.ts

```diff
@@ -0,0 +1,5 @@
+import type { Document } from './document'
+
+export type UISheet = {
+  documents: Document[]
```

Comment:
would it make sense to associate each document with a key of some sort, for easier reverse lookup?

Reply:
Unresolving because yea we should map it, similar to current UI sheet structure